server/daos/books.js

```diff
@@ -15,3 +15,54 @@ export const updateBook = async args => {
 
   return book;
 };
+
+export const queryUsingLanguage = async args => {
+  let affectedRows;
+  try {
+    affectedRows = await db.books.findAll({
+      include: [
+        {
+          model: db.languages,
+          through: { where: { language: args.language } }
+        }
+      ]
+    });
+    console.log('affected rows', affectedRows);
+  } catch (e) {
+    throw new Error(`Failed to find the books written in ${args.language}`);
+  }
+  if (!affectedRows) {
+    throw new Error('Data not found');
```

Comment:
i think we went over this one. we can't be throwing an error if we don't find data